go.mod

```diff
@@ -76,3 +76,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/lawyerphx/parser v0.0.0-20200316023126-7af13c9e0b85
```

Comment:
Please remove this.

Reply:
OK， waiting for this PR to be merged (#775)[https://github.com/pingcap/parser/pull/775]